sat-plugin/src/main/java/org/openhab/tools/analysis/report/ReportMojo.java

```diff
@@ -115,6 +117,18 @@ public class ReportMojo extends AbstractMojo {
     @Parameter(property = "report.fail.on.error", defaultValue = "true")
     private boolean failOnError;
 
+    /**
+     * Describes of the build should fail if medium priority error is found
+     */
+    @Parameter(property = "report.fail.on.warning", defaultValue = "false")
+    private boolean failOnWarning;
+
+    /**
+     * Describes of the build should fail if low priority error is found
+     */
+    @Parameter(property = "report.fail.on.debug", defaultValue = "false")
+    private boolean failOnDebug;
```

Comment:
Is there a reason why you opted for calling this `debug` instead of `info`?

The output uses info for this: 

```
[INFO] --- sat:0.17.0:report (sat-all) @ org.openhab.core.thing.tests ---
[WARNING] Code Analysis Tool has found: 
 0 error(s)! 
 1 warning(s) 
 0 info(s)
```

Reply:
No particular reason, I updated this to be "info" as it probably makes more sense.